temporal-serviceclient/build.gradle

```diff
@@ -77,19 +77,21 @@ dependencies {
     errorproneJavac('com.google.errorprone:javac:9+181-r4173-1')
     errorprone('com.google.errorprone:error_prone_core:2.5.1')
 
-    api 'io.grpc:grpc-protobuf:1.35.0'
-    api 'io.grpc:grpc-stub:1.35.0'
-    api 'io.grpc:grpc-core:1.35.0'
-    api group: 'com.google.protobuf', name: 'protobuf-java-util', version: '3.14.0'
-    api group: 'com.uber.m3', name: 'tally-core', version: '0.6.1'
+    api 'io.grpc:grpc-protobuf:1.36.0'
```

Comment:
I suggest making dependency updates in the separate PR.
This way if something goes wrong problems will be easier to locate and roll back.